kaggle/cli.py

```diff
@@ -1674,11 +1674,11 @@ class Help(object):
     command_model_instances_new = 'Create a new model instance'
     param_model_instance_downfile = (
         'Folder for downloading the special model-instance-metadata.json file '
-        '(https://github.com/Kaggle/kaggle-api/wiki/ModelInstance-Metadata). ')
+        '(https://github.com/Kaggle/kaggle-api/wiki/Model-Metadata#model-instance). ')
```

Comment:
I see a bunch of models changes that seems unrelated to your PR. Can you make sure to rebase it to make it easier to review? TY

Reply:
`Current branch convert-comps is up to date.`

I had some `git` problems and ended up including commits that have already been merged.

Reply:
Not sure the exact `git` problems you are running into given I don't know your `git` state and which command you ran but your could try the following:

1. Squash all your commits into one: https://stackoverflow.com/questions/5189560/how-do-i-squash-my-last-n-commits-together (this is to make step 2 easier. Simpler to only rebase one commit).
1. Rebase against main: `git rebase origin main`

Reply:
These changes are no longer included.